tip-wlan/charts/opensync-gw-cloud/values.yaml

```diff
@@ -121,6 +121,13 @@ externalhostaddress:
   ovsdb: opensync-gw-cloud
```

Comment:
I would group the externalhost address and ports together under `externalhost`.
So something like:
```
externalhost:
   address:
      ovsdb:
      mqtt:
   ports:
      ovsdb:
      mqtt:
```

Reply:
Thank you @rahulsharma2020, will make the change